src/index.js

```diff
@@ -9,14 +9,12 @@ import App from './App';
 import './index.css';
 import reducer from './reducers';
 
-const store = createStore(
-  reducer,
-  compose(
-    applyMiddleware(thunk),
-    window.__REDUX_DEVTOOLS_EXTENSION__ &&
-      window.__REDUX_DEVTOOLS_EXTENSION__(),
-  ),
-);
+const enhancer = window.__REDUX_DEVTOOLS_EXTENSION__ &&
```

Comment:
How about:
```
let storeEnhancer = [applyMiddleware(thunk)];

if (window.__REDUX_DEVTOOLS_EXTENSION__ &&
    process.env.NODE_ENV === 'development') {
  storeEnhancer = [...storeEnhancer, window.__REDUX_DEVTOOLS_EXTENSION__()];
}

const store = createStore(reducer, compose(...storeEnhancer));
```
?

It let us add and remove enhancer without changing a lot of lines.
I also think it is easier to read, as the first and default enhancer is in the first line and every additional enhancer gets their own lines.